base_layer/core/src/blocks/pre_mine/mod.rs

```diff
@@ -804,9 +804,12 @@ pub async fn create_pre_mine_genesis_block_info(
         .enumerate()
     {
         let signature_threshold = get_signature_threshold(public_keys.len())?;
-        let total_script_key = public_keys.iter().fold(PublicKey::default(), |acc, x| acc + x);
+        let mut total_script_key = RistrettoPublicKey::default();
```

Comment:
With the proposed change to types ^ we can continue to use `PublicKey`
```suggestion
        let mut total_script_key = PublicKey::default();
```

Reply:
I dont want to use public key at all, as it has a conflict with the trait name